inst/pkglock_test.Rmd

```diff
@@ -0,0 +1,78 @@
+---
+title: "pkglock test"
+author: "Brian Schultheiss"
+date: "April 11, 2018"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# Setup installer
+
+```{r }
+
+#Setup Installer
+installerdir <- file.path( getwd(),"pkglock/inst/installreq")
+dir.create(installerdir, showWarnings = FALSE, recursive = TRUE)
+devtools::install_github("bschulth/pkglock", lib=installerdir, force=TRUE, dependencies=TRUE)  
+
+#This seems to be needed also, but is not getting pulled in by default
+install.packages("remotes", lib = installerdir)
+.libPaths( c( .libPaths(), installerdir) )
+
+#List installed packages in installerdir
+pkglock::build_package_list_from_library_dir(installerdir)
+#hmm...install_github() doesn't seem to respect lib=installerdir arg
+
+#hmm....seem to have to force load this here before calling rest of pkglock functions otherwise 

```

Comment:
I'm not sure why this behavior is happening.  What happens if not?

Reply:
maybe this was happening because I was using the 
     pkglock::xxxxxx 
invocation rather than:
     library(pkglock)
     xxxxxx